concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -57,7 +57,7 @@ update_pipeline() {
         <(generate_vars_file)
     ;;
     autodelete-cloudfoundry)
-      if [ ! "${DISABLE_AUTODELETE:-}" ]; then
+      if [ "${ENABLE_AUTODELETE:-}" = "true" ]; then
```

Comment:
:+1: inverting this.

Though I think the previous behaviour of accepting any set value is preferable. I realise it allows inconsistencies in what values can be set, but it does mean that people don't have to hunt out the conditional to see what a valid value is. And it would be consistent with the behaviour of `ENABLE_AUTO_DEPLOY`.